src/toolbars/examples/toolbar.js

```diff
@@ -119,6 +119,7 @@
       <div class="col-md-12" ng-show="viewType == 'tableView'">
         <pf-table-view config="tableConfig"
                        colummns="colummns"
+                       colvis="colvis"
                        items="items"
```

Comment:
First, $scope.colummns is special for defining table head. Second, "colvis" is just an intermediate variable which is in charge of telling pfTableView which column's visibility is toggled. Any time, there is only one colvis variable.